src/constants.js

```diff
@@ -361,8 +361,7 @@ export const RESOURCE_PAGE_PATH =
 export const PLAIN_ARTICLE_PAGE_PATH = '/article/:articleId';
 export const SEARCH_PATH = '/search(.*)';
 export const TOPIC_PATH = '/subjects/:subjectId/:topicPath(.*)?/:topicId';
-export const SUBJECT_PAGE_PATH =
-  '/subjects/:subjectId/:topicId?/:subTopicId?/:subSubTopicId(!resource)?/';
+export const SUBJECT_PAGE_PATH = '/subjects/:subjectId/:topics*/';
```

Comment:
Klarte ikkje finne en path som utelukker resources uavhengig av nivå. React-router har ikkje støtte for avansert regexp som kunne løst dette.